src/neuroglancer/ui/annotations.ts

```diff
@@ -1342,12 +1347,30 @@ export function UserLayerWithAnnotationsMixin<TBase extends {new (...args: any[]
                                                   }))),
                   ({annotation, chunkTransform}, parent, context) => {
                     if (annotation == null) {
-                      const statusMessage = document.createElement('div');
-                      statusMessage.classList.add('neuroglancer-selection-annotation-status');
-                      statusMessage.textContent =
-                          (annotation === null) ? 'Annotation not found' : 'Loading...';
-                      parent.appendChild(statusMessage);
-                      return;
+                      if (state.annotationType && state.annotationSerialized) {
```

Comment:
Check `state.annotationType !== undefined && state.annotationSerialized !== undefined`.

Otherwise the check will fail for AnnotationType.POINT which happens to have a value of 0.

Reply:
Updated.